FAQ.md

```diff
@@ -201,6 +201,26 @@ We'd love to consume machine-readable numbering plan data (assigned ranges,
 carrier & geo mappings). If you can connect us with partners in the industry
 to achieve this, please do so. Thanks!
 
+### Why is this number from Argentina (AR) or Mexico (MX) not identified as the right number type?
+
+Certain countries' mobile and/or fixed line ranges may overlap, which may make
+accurate identification impossible without additional and explicit context.
+For example, when calling a mobile line from a fixed line in Argentina,
+you need to dial 15 before the subscriber number, or 9 if you're calling
+from another country. Without these additional digits, you may end up calling
+a different endpoint belonging to a fixed line!
```

Comment:
This last part isn't true: because I don't think (at least anymore) that you have cases where ABCDEF is assigned to fixed-line and 9 ABCDEF is assigned to mobile. It's just that it doesn't work without the additional digits...